source/release-notes/2.4.0.rst

```diff
@@ -0,0 +1,63 @@
+* :jirabug:`PMM-5329`: Prometheus in PMM Server was updated to version 2.16.0. This update has brought several improvements. Among them are significantly reduced memory footprint of the loaded TSDB blocks, lower memory footprint for the compaction process (caused by the more balanced choice of what to buffer during compaction), and improved query performance for the queries that only touch the most recent 2h of data.
+* :jirabug:`PMM-5210`: Data Retention is now specified in days instead of seconds on the PMM Settings page. Please note this is the UI-only change, so the actual data retention precision is not changed
+* :jirabug:`PMM-5182`: The logs.zip archive available on the PMM Settings page now includes additional self-monitoring information in a separate "client" subfolder. This subfolder contains information collected on the PMM Server and is equivalent to the one collected on a node by the ``pmm-admin --summary`` command. 
+* :jirabug:`PMM-4915`: The Query Analytics dashboard now shows Time Metrics in the Profile Section as "AVG per query" instead of "AVG per second"
+* :jirabug:`PMM-5112`: The Inventory API List requests now can be filtered by the Node/Service/Agent type
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PMM-5178`: Query Detail Section of the Query Analytics dashboard didn't show tables definitions and indexes for the internal PostgreSQL database
+* :jirabug:`PMM-5465`: MySQL Instance related dashboards had row names not always matching the actual contents. To fix this, elements were re-ordered and additional rows were added for better matching of the row name and the corresponding elements
+* :jirabug:`PMM-5455`: Dashboards from the Insight menu were fixed to work correctly when the low resolution is set on the PMM Settings page
+* :jirabug:`PMM-5446`: A number of the Compare Dashboards were fixed to work correctly when the low resolution is set on the PMM Settings page
+* :jirabug:`PMM-5445`: Labels change (which occurs e.g. when the metrics resolution is changed on the PMM Settings page) was breaking the HA Dashboards
+* :jirabug:`PMM-5439`: Labels change (which occurs e.g. when the metrics resolution is changed on the PMM Settings page) was breaking the Dashboards for the RDS instances
+* :jirabug:`PMM-5430`: MySQL Exporter section on the Prometheus Exporter Status dashboard now collapsed by default to be consistent with other database-related sections
+* :jirabug:`PMM-5427`: Labels change (which occurs e.g. when the metrics resolution is changed on the PMM Settings page) was breaking the PostgreSQL dashboards
+* :jirabug:`PMM-5426`: Labels change (which occurs e.g. when the metrics resolution is changed on the PMM Settings page) was breaking the MongoDB dashboards
+* :jirabug:`PMM-5419`: Labels change (which occurs e.g. when the metrics resolution is changed on the PMM Settings page) was breaking the MySQL dashboards
```

Comment:
I think everything (except PMM-5430) can be grouped into one description because this is the same change to different dashboards.